src/clojure/nrepl/middleware/interruptible_eval.clj

```diff
@@ -4,6 +4,7 @@
    clojure.main
    [nrepl.middleware :refer [set-descriptor!]]
    nrepl.middleware.pr-values
+   nrepl.middleware.truncate-values-and-outputs
```

Comment:
I guess we can shorten the name to simply `truncate`, but that's not a big deal.